CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publicationt. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
+|Identifier|identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier|Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Contact|extraProperties - contact|**RECOMMENDED**| Contact information for this dataset.|Provide contact information (name and email address) of the person responsible for the dataset.| 
+|Logo|logo|**RECOMMENDED**| Link to or provide a logo to display on your resource landing page.|Link to an URL for the logo or local filename containing the logo.
+|Dates|dates|**OPTIONAL**|  Relevant dates for the dataset. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first data collection, last data collection, date of first publication, ...) |<!--will later choose from a pulldown list--> 
+|Produced By|producedBy|**OPTIONAL**| Process which generated a given dataset.|Process which generated a given dataset.|
+|Is About|isAbout|**OPTIONAL**| Entities (biological entity, taxonomic information, disease, molecular entity, anatomical part, treatment) associated with this dataset.| |
+|Acknowledges|acknowledges|**OPTIONAL**| Grant(s) which funded and supported the work reported by the dataset.| Grant(s) which funded and supported the work reported by the dataset.|
+|Spatial Coverage|spatialCoverage|**OPTIONAL**| The geographical extension and span covered by the dataset.|The geographical extension and span covered by the dataset.|
```

Comment:
I don't understand the term 'geographical extension and span covered'.

Reply:
This is relevant to datasets where the location of the original subjects is relevant.  For example if a research group in Ottawa carried out a study on data from patients in Montreal, this is where we would store the value "Montreal".

Reply:
I will add to the help file "i.e.: city, province, administrative region, ...)"